### ISSUE TITLE
Better kill tally behavior

### ISSUE THREAD

**Issue by Blastfrog** [team-eternity/eternity#115]
When the percentage of kills is calculated, it should take into account newly spawned monsters, creating a new total to compare against instead of what the map started with. For example, if a dehacked patch was loaded that made lost souls count as kills, any new lost soul added to the map by a pain elemental would increase the total that the tally uses. This way, you'd never go over 100% kills, and any lingering lost souls would cause the tally to be less than 100%.

Needless to say, there should be a compatibility option to use the original behavior.

**Comment by haleyjd** [comment]
I disagree; kill count behavior should remain like in vanilla. However supporting a separate living enemies count like PrBoom+ does is a better idea and one that's been on my "list" for a long time.

**Comment by Blastfrog** [comment]
Could it still be an option to handle it like I described? With the default being vanilla's behavior, of course.
